src/livekit/openIDSFU.ts

```diff
@@ -55,13 +61,28 @@ export function useOpenIDSFU(
         },
       );
     } else {
+      currentFocus.current = {} as LivekitFocus;
       setSFUConfig(undefined);
     }
   }, [client, activeFocus, showErrorBoundary]);
```

Comment:
Using the primitive types should also maek this work:
```suggestion
  }, [client, activeFocus.type, activeFocus.livekit_service_url, activeFocus.livekit_alias, showErrorBoundary]);
```
We might need to then recompose the active focus to make eslint happy:
```
getSFUConfigWithOpenID(client,
    {
      type: activeFocus.type,
      livekit_service_url: activeFocus.livekit_service_url,
      livekit_alias: activeFocus.livekit_alias
      }
     ).then(...
```

But we dont need the ref and the comparison function anymore.